source/components/GlobalNavigation/components/GlobalNavigationNotifications/Card.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Timeago from '../../../Timeago';
+
+import { isAnnouncement, isDiscussionReply } from '../../models/notificationTypes';
+
+import MarkAsReadIcon from './MarkAsReadIcon';
+import CardText from './CardText';
+import CardLink from './CardLink';
+
+const Card = ({ model, track }) => {
+    const { totalUniqueActors, isUnread, type, title, latestActors, timestamp, communityName, snippet } = model;
+    const computedClass = classNames('wds-notification-card', isUnread && 'wds-is-unread');
```

Comment:
Minor: How about updating this to:
```js
const computedClass = classNames('wds-notification-card', {'wds-is-unread': isUnread});
```
?
Using an object with booleans to determine if the classname should be included is how the classnames docs suggest doing it: https://www.npmjs.com/package/classnames#usage